apps/web/components/eventtype/EventAdvancedTab.tsx

```diff
@@ -106,23 +106,24 @@ export const EventAdvancedTab = ({ eventType, team }: Pick<EventTypeSetupProps,
     );
   };
 
-  const { shouldLockDisableProps } = useLockedFieldsManager(
-    formMethods.getValues(),
-    t("locked_fields_admin_description"),
-    t("locked_fields_member_description")
-  );
+  const { isChildrenManagedEventType, isManagedEventType, shouldLockDisableProps, shouldLockIndicator } =
+    useLockedFieldsManager(eventType, formMethods, t);
   const eventNamePlaceholder = getEventName({
     ...eventNameObject,
     eventName: formMethods.watch("eventName"),
   });
 
   const successRedirectUrlLocked = shouldLockDisableProps("successRedirectUrl");
   const seatsLocked = shouldLockDisableProps("seatsPerTimeSlotEnabled");
+  const requiresBookerEmailVerificationProps = shouldLockDisableProps("requiresBookerEmailVerification");
+  const hideCalendarNotesLocked = shouldLockDisableProps("hideCalendarNotes");
+  const lockTimeZoneToggleOnBookingPageLocked = shouldLockDisableProps("lockTimeZoneToggleOnBookingPage");
 
   const closeEventNameTip = () => setShowEventNameTip(false);
   const displayDestinationCalendarSelector =
     !!connectedCalendarsQuery.data?.connectedCalendars.length && !team;
```

Comment:
Would this also be a good opprotunity to allow children event types to select their own destination calendar?

Reply:
Yeah, good spot. I think this makes sense!